### PR TITLE
Updating docs for global install usage

### LINKED ISSUE #16: don't make us require it in the app
I suggest we find a way not to require it in index.js (you do suggest to install it as dev dependency which would mean it is not available in production) 

possible solutions:

* wrap it in a logical test on the environement >>  not ideal, and bloat
* wrap it in a separate dev.index.js which then loads index.js
* ...

### PR DESCRIPTION
**Pull request by jasonwilson**
Update the docs to include examples and patterns for global install. 

@ryan-roemer 

Closes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,26 +10,50 @@
...
 #### Add the reporting module to your code
 
-From within your `index.js` or app entry point simply require the `nodejs-dashboard` module.
+There are a couple patterns for including `nodejs-dashboard` in your code. Since this module isn't designed for production use it's best to conditionally include.
+
+##### Including via preload argument
+
+This pattern requires you to use a run-time flag (see below).
+
+`nodejs-dashboard -- node -r nodejs-dashboard index.js`
```
Comment (aisapatino) [inline README.md]: What does the `-r nodejs-dashboard` do? I know it requires the module, but for me that just results in the nodejs-dashboard usage info being printed out at the start. `nodejs-dashboard node index.js` would work too, right?

> requires you to use a run-time flag (see below)
Does the "see below" refer to npm script example?

Comment (jasonwilson) [inline README.md]: It requires the module from the command-line so it doesn't need to be included in code.

**Comment by aisapatino** [comment]
lgtm!

File: README.md (hunk 1 above)
Comment (ryan-roemer) [inline README.md]: I think there is potentially confusing part is the double occurrence of the string `nodejs-dashboard`. Maybe add a sentence or two to say something like:

"This is equivalent to doing

```sh
nodejs-dashboard -- node index.js
```

and adding the line:

```js
require("nodejs-dashboard");
```

At the top of `index.js`"

?

File: README.md [hunk 2]
```diff
@@ -10,26 +10,50 @@
...
 
-It's recommended that you create a npm script to launch the dashboard.
+It's recommended that you create a npm script to launch the dashboard. How you decide to include the module will dictate the run script.
+
+For example, if you want to conditionally include via requires flag see the following example.
+```
+...
+"scripts": {
+    "dev": "nodejs-dashboard -- node -r nodejs-dashboard index.js"
+  }
```
Comment (ryan-roemer) [inline README.md]: GLOBAL NIT: `"scripts"` and closing brace `}` throughout this whole doc seem misaligned.

File: README.md [hunk 3]
```diff
@@ -10,26 +10,50 @@
...
 require("nodejs-dashboard");
 ```
 
 #### Update your package.json
 
-It's recommended that you create a npm script to launch the dashboard.
+It's recommended that you create a npm script to launch the dashboard. How you decide to include the module will dictate the run script.
+
+For example, if you want to conditionally include via requires flag see the following example.
+```
```
Comment (ryan-roemer) [inline README.md]: GLOBAL NIT: `js` syntax highlighting?

File: README.md (hunk 1 above)
Comment (ryan-roemer) [inline README.md]: Separate NIT: Make into fences and real shell command instead of single ticks?

**Comment by ryan-roemer** [comment]
@jasonwilson -- Great work! LGTM

**Pull request merged**